src/speculators/cli.py

```diff
@@ -0,0 +1,37 @@
+"""
```

Comment:
Migrate to \_\_main__.py

https://github.com/neuralmagic/guidellm/blob/main/pyproject.toml#L111:

[project.entry-points.console_scripts]
guidellm = "guidellm.\_\_main__:cli"